.github/workflows/snap.yaml

```diff
@@ -15,26 +16,12 @@ on:
         default: ''
 
 jobs:
-  snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
-    secrets:
-      store-login: ${{ secrets.STORE_LOGIN }}
-    with:
-      ubuntu-image: ubuntu-22.04
-      branch-name: ${{ inputs.branch-name == '' && github.ref || inputs.branch-name }}
-      snapcraft-build-path: ./turtlebot3c-bringup-snap
-      snapcraft-args: --enable-experimental-extensions
-      snap-name: turtlebot3c-bringup
-      publish: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
-      publish-track: humble
-
-  model-snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
+  snaps:
+    uses: canonical/robotics-actions-workflows/.github/workflows/snap.yaml@main
     secrets:
-      store-login: ${{ secrets.STORE_LOGIN }}
+      snapstore-login: ${{ secrets.STORE_LOGIN }}
     with:
-      ubuntu-image: ubuntu-22.04
-      branch-name: ${{ inputs.branch-name == '' && github.ref || inputs.branch-name }}
-      snapcraft-build-path: ./turtlebot3c-model-snap
-      snap-name: turtlebot3c-model
-      publish: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
+      runs-on: '["ubuntu-latest", ["self-hosted", "linux", "ARM64", "medium", "noble"]]'
+      snapcraft-source-subdir: "turtlebot3c-bringup-snap"
+      git-ref: ${{ inputs.branch-name == '' && github.ref || inputs.branch-name }}
+      snapcraft-enable-experimental-extensions: true
```

Comment:
The Humble extension [isn't experimental](https://github.com/canonical/snapcraft/blob/187187d53d735d6f19dac7f474f321bd4c467f08/snapcraft/extensions/ros2_humble.py#L47) anymore.